### DISCUSSION TITLE
Easy token price estimates for 400+ LLMs

### DISCUSSION THREAD

**Discussion by derevyan** [langfuse/langfuse#2493] [Ideas]
### Describe the feature or potential improvement

Please check this out - https://github.com/AgentOps-AI/tokencost  maybe can be helpful to add pricing calculating and the estimated cost for all models, like gemini. e.t.c

**Comment by marcklingen** [comment]
This is a great repo, Langfuse runs on Typescript, but we plan to modularize tokenization and pricing information to make it easier to support more models by default.
